analyzers/src/ITs.JsonParser/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using CommandLine;
+using ITs.JsonParser;
+
+Parser.Default.ParseArguments<CommandLineOptions>(args).MapResult(Execute, errs => 1);
+
+int Execute(CommandLineOptions options)
+{
+    WriteLineColor("Processing analyzer results", ConsoleColor.Yellow);
+    NewIssueReports();
+    ShowDiffResults(options.Project, options.RuleId);
+    return 0;
+}
+
+// TODO: Reimplement this functionality from the powershell function.
+void NewIssueReports()
+{
+    Console.WriteLine("Normalizing the SARIF reports");
+}
+
+// TODO: Reimplement this functionality from the powershell function.
+void ShowDiffResults(string project, string ruleId)
+{
+    // Placeholder, to be deleted
+    if (project is not null)
+    {
+        Console.WriteLine($"Target Project: {project}");
+    }
+    if (ruleId is not null)
+    {
+        Console.WriteLine($"Target Rule: {ruleId}");
+    }
+}
+
+void WriteLineColor(string value, ConsoleColor color)
+{
+    var before = Console.ForegroundColor;
+    Console.ForegroundColor = color;
+    Console.WriteLine(value);
+    Console.ForegroundColor = before;
+}
+
+
+// TODO: Fix this stupid hack for implicit global usings.
+namespace Microsoft.CodeAnalysis { }
+namespace Microsoft.CodeAnalysis.Diagnostics { }
```

Comment:
This is very stupid, but I don't know how else to work around it.
This [targets file](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/Analyzers.targets#L24) is adding an implicit global using to every project below it, so the project cannot be built without these namespaces. 
Any suggestions? :hide-the-pain: